source/class/cv/ui/structure/pure/Slide.js

```diff
@@ -112,7 +112,7 @@ qx.Class.define('cv.ui.structure.pure.Slide', {
         slider.init();
         // set initial value
         slider.setValue(parseFloat(this.getValue()));
-        var throttled = qx.util.Function.throttle(this._onChangeValue, 250, {trailing: true}).bind(this);
+        var throttled = qx.util.Function.throttle(this._onChangeValue, 250, {trailing: false}).bind(this);
```

Comment:
`{trailing: true}` (or something similar?) is required to not break the user action:

When dragging the slider quickly out of it's range (e.g. to force it to 0% or 100%) will not work anymore with this change